pkgs/tools/security/fail2ban/default.nix

```
@@ -71,6 +76,8 @@ python3.pkgs.buildPythonApplication rec {
     ''
       # see https://github.com/NixOS/nixpkgs/issues/4968
       rm -r "${sitePackages}/etc"
+
+      installManPage man/fail2ban.1 man/jail.conf.5 man/fail2ban-{client,server,regex,python,testcases}.1
```

```suggestion
      installManPage man/*.?
```

How about just including all single character extension files.

---

Seems like all manpages are consistently named throughout the previous versions and not all of the versions have the same files so this seems a better way to get the manpages into the output. Although I went with `man/*.[1-9]` as the glob to make it clearer in the code.